### PR TITLE
doc: release-notes: LoRaWAN notes and zbus fixup

### PR DESCRIPTION
**Pull request by JordanYates**
LoRaWAN release notes and a fix on the zbus notes.

### CODE REVIEW

File: doc/releases/release-notes-3.6.rst [hunk 1]
```diff
@@ -488,14 +488,17 @@ Libraries / Subsystems
...
 * ZBus
 
-  * Renamed :kconfig:option:`ZBUS_MSG_SUBSCRIBER_NET_BUF_DYNAMIC` and
-    :kconfig:option:`ZBUS_MSG_SUBSCRIBER_NET_BUF_STATIC`
-    with :kconfig:option:`ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC` and
-    :kconfig:option:`ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_STATIC`
+  * Renamed :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_NET_BUF_DYNAMIC` and
+    :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_NET_BUF_STATIC`
+    with :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC` and
+    :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_STATIC`
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-3.6.rst:501]: The old values should be enclosed in double backticks.

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.6.rst (hunk 1 above)
Comment (kartben) [reply doc/releases/release-notes-3.6.rst]: and this probably rather belongs to migration guide anyway, right?

Comment (JordanYates) [reply doc/releases/release-notes-3.6.rst]: I was just fixing up incorrect usage of `:config:option:`, not adding this section.

**Review by MaureenHelm — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Comment by MaureenHelm** [comment]
@henrikbrixandersen can you take another look?

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**